packages/core/test-support/helpers/contains-helpers.js

```diff
@@ -0,0 +1,29 @@
+import { findAll } from '@ember/test-helpers';
+
+/**
+ * Finds selector by innerText
+ * @param {String} selector - selector to search
+ * @param {String} text - text to search by
+ * @returns {Element} - first matching dom element
+ */
+export default function findByContains(selector, text) {
+  return findAll(selector).filter(el => el.innerText.includes(text))[0];
+}
+
+/**
+ * Finds button by innnerText
+ * @param {String} text - text to search for
+ * @returns {Element} - first matching dom element
+ */
+export function button(text) {
```

Comment:
Maybe `button` and `link` would be better with more descriptive names. They make sense in the scope of this file, but anywhere it's imported to wouldn't necessarily have the context to tell what these functions do.